cmp/tab/TabModel.js

```diff
@@ -44,7 +44,7 @@ export class TabModel extends HoistModel {
      *      but still be able to activate the tab manually or via routing.
      * @param {boolean} [c.showRemoveAction] - display an affordance to allow the user to remove
      *      this tab from its container.
-     * @param {(Object|function)} c.content - Hoist Component (class or functional) to be rendered by this
+     * @param {(Element|Object|function)} c.content - Hoist Component (class or functional) to be rendered by this
```

Comment:
We could leave `Element` this out of the signature, even if supported generally by the util, we don't need to declare it supported (in the sense of recommended in a documented way) here.

I also think that `Element` resolves to the DOM object, not the react one - we should confirm - might need to bomb the code with `{ReactElement}` instead, but I'd want to look into it before doing so to ensure I've got that right.